### ISSUE TITLE
Wrong Output Problem 

### ISSUE THREAD

**Issue by imamim** [MaybeShewill-CV/lanenet-lane-detection#487]
**First of all, thank you for your work.**

I processed the picture from a video I found on the Internet in lane-net model. Below is the Output I obtained:

![Ekran Görüntüsü - 2021-02-13 13-46-42](https://user-images.githubusercontent.com/71751858/107850989-833c4380-6e17-11eb-80b2-69a2e05542e5.png)

The model gave me a wrong result here, I also get the wrong result in the same way in the 1-2 different pictures I have tried.

I did my experiment by running the "_test_lanenet.py_" file.

Lanes appear to have been detected in the "_Instance Image_", but the last output "_mask_" and "_src_" seems to be quite a distant result.

What do you think is the reason for this?

I am waiting for your help, thanks in advance !!

**Comment by imamim** [comment]
> **First of all, thank you for your work.**
> 
> I processed the picture from a video I found on the Internet in lane-net model. Below is the Output I obtained:
> 
> ![Ekran Görüntüsü - 2021-02-13 13-46-42](https://user-images.githubusercontent.com/71751858/107850989-833c4380-6e17-11eb-80b2-69a2e05542e5.png)
> 
> The model gave me a wrong result here, I also get the wrong result in the same way in the 1-2 different pictures I have tried.
> 
> I did my experiment by running the "_test_lanenet.py_" file.
> 
> Lanes appear to have been detected in the "_Instance Image_", but the last output "_mask_" and "_src_" seems to be quite a distant result.
> 
> What do you think is the reason for this?
> 
> I am waiting for your help, thanks in advance !!

**Comment by imamim** [comment]
After giving the image (1280,720) in size, I got the following result:

![sorun](https://user-images.githubusercontent.com/71751858/108089121-4b890200-708a-11eb-9a7e-2f00735dbba4.png)

Although I noticed some of my mistakes, there still seems to be a problem with the output. While the correct results appear in "_Instance Image_" and "_binary_image_", there is still a single lane in "_mask_image_". I think this loss occur after the clustering.

I just wanted to inform.

By the way, do not mind my inexperience :)

Waiting for your help, thank you!
